Actions n dreplacements

kaitokidi opened the pull request:
traducció de actions.json

+ alguns canvis generals de find and replace



+ aquestes són els noms que s'han canviat per si s'hi fa referència en alguna altre banda del codi, també caldria fer search and replace 
@action Activate an Item @action Activar un Objecte
@action Attack @action Atacar
@action Cast a Spell @action Fer un conjur
@action Climb onto a Bigger Creature @action Enfilar-se a una criatura més gran
@action Dash @action Córrer
@action Disarm @action Desarmar
@action Disengage @action Destravar-se
@action Dodge @action Esquivar
@action Don or Doff a Shield @action Equipar o desequipar un Escut
@action End Concentration @action Acabar la concentració
@action Escape a Grapple @action Escapar una presa
@action Grapple @action Apresar
@action Healing Surge @action Esforç Curatiu
@action Help @action Ajudar
@action Hide @action Amagar-se
@action Identify a Spell @action Identificar un conjur
@action Improvising an Action @action Improvisar una Acció
@action Mark @action Marcar
@action Opportunity Attack @action Atac d'oportunitat
@action Other Activity @action Altres activitats
@action Overrun @action Sobrepassar
@action Ready @action Preparada
@action Search @action Cerca
@action Empenta @action Empenta
@action Shove Aside @action Apartar
@action Tumble @action Malabarismes
@action Two-Weapon Fighting @action Lluita amb dues armes
@action Use an Object @action Utilitzar un Objecte

kaitokidi committed:
actions and replacements

kaitokidi committed:
actions translated

daniel-giralt-len commented on data/actions.json:
```diff
@@ -1,7 +1,8 @@
 	"action": [
 		{
-			"name": "Activate an Item",
+			"name": "Activar un ítem",
```
Si prefereixes ho podem canviar a Activar un Objecte, que tenies tota la raó sona molt millor!

daniel-giralt-len commented on data/actions.json:
```diff
@@ -12,42 +13,42 @@
 			"entries": [
-				"Activating some magic items requires a user to do something in particular, such as holding the item and uttering a paraula màgica, reading the item if it is a scroll, or drinking it if it is a potion. The description of each item category or individual item details how an item is activated. Certain items use one or more of the following rules related to their activation.",
-				"If an item requires an action to activate, that action isn't a function of the {@action Use an Object} action, so a feature such as the rogue's {@subclassFeature Fast Hands|Rogue||Thief||3} can't be used to activate the item.",
+				"L'activació d'alguns objectes màgics requereix que l'usuari faci alguna cosa en particular, com ara aguantar-lo i pronunciar una paraula màgica, llegir-lo si es tracta d'un rotllo o beure'l si és una poció. La descripció de cada categoria d'ítem o ítem en sí, detalla com s'activa. Alguns ítems fan servir una o més de les regles següents relacionades amb la seva activació.",
```
```suggestion
				"L'activació d'alguns objectes màgics requereix que l'usuari faci alguna cosa en particular, com ara aguantar-lo i pronunciar una paraula màgica, llegir-lo si es tracta d'un pergamí o beure'l si és una poció. La descripció de cada categoria d'ítem o ítem en sí, detalla com s'activa. Alguns ítems fan servir una o més de les regles següents relacionades amb la seva activació.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -12,42 +13,42 @@
 					"name": "Consumibles",
 					"entries": [
-						"Some items are used up when they are activated. A potion or elixir must be swallowed, or an oil applied to the body. The writing vanishes from a scroll when it is read. Once used, a consumable item loses its magic and no longer functions."
+						"Alguns ítems es gasten quan s'activen. Una poció o elixir, s'han d'empassar, o un oli s'ha d'aplicar al cos. L'escriptura desapareix d'un rotllo quan es llegeix. Un cop utilitzat, un ítem consumible perd la seva màgia i la seva funcionalitat."
```
```suggestion
						"Alguns ítems es gasten quan s'activen. Una poció o elixir, s'han d'empassar, o un oli s'ha d'aplicar al cos. L'escriptura desapareix d'un pergamí quan es llegeix. Un cop utilitzat, un ítem consumible perd la seva màgia i la seva funcionalitat."
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -12,42 +13,42 @@
-						"Some magic items allow the user to cast a spell from the item, often by expending charges from it. The spell is cast at the lowest possible spell and caster level, doesn't expend any of the user's spell slots, and requires no components unless the item's description says otherwise. The spell uses its normal casting time, range, and duration, and the user of the item must concentrate if the ...
-						"Many items, such as potions, bypass the casting of the spell and confer the spell's effects. Such an item still uses the spell's duration unless the item's description says otherwise.",
-						"A magic item, such as certain staffs, may require you to use your own habilitat de llançament de conjurs when you cast a spell from the item. If you have more than one habilitat de llançament de conjurs, you choose which one to use with the item. If you don't have a habilitat de llançament de conjurs\u2014perhaps you're a rogue with the Use Magic Device feature\u2014el teu modificador de l ...
+						"Alguns objectes màgics permeten a l'usuari llançar un conjur de l'ítem, sovint gastant-ne càrregues. El conjur es llança al nivell més baix possible de conjur i llançador, no gasta cap de les ranures de conjur de l'usuari i no requereix components tret que a la descripció de l'ítem hi digui el contrari. El conjur, utilitza el seu temps, abast i durada normals de llançament, i l'usuari de l ...
```
```suggestion
						"Alguns objectes màgics permeten a l'usuari llançar un conjur de l'ítem, sovint gastant-ne càrregues. El conjur es llança al nivell més baix possible de conjur i llançador, no gasta cap de les ranures de conjur de l'usuari i no requereix components tret que a la descripció de l'ítem hi digui el contrari. El conjur, utilitza el seu temps, abast i durada normals de llançament, i l'usuari de l'ítem s'ha de concentrar si el conjur requereix concentració. Alguns ítems fan excepcions a aquestes regles, canviant el temps, la temps de llançament o la duració del conjur, o altres parts del conjur",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -12,42 +13,42 @@
 		},
 		{
-			"name": "Attack",
+			"name": "Atac",
```
Li he anat posant Atacar, per mantenir aquest infinitiu de les accions (Utilitzar un objecte, Destravar-se)

daniel-giralt-len commented on data/actions.json:
```diff
@@ -58,50 +59,50 @@
-				"With this action, you make one melee or ranged attack. See the \"{@book Making an Attack|phb|9|making an attack}\" section for the rules that govern attacks.",
-				"Certain features, such as the Extra Attack feature of the fighter, allow you to make more than one attack with this action."
+				"L'acció més freqüent en combat és l'acció d'atac, tant si s'esgrimeix una espasa, es dispara una fletxa amb un arc o es dona un cop de puny.",
+				"Amb aquesta acció, feu un atac a melé o a distància. Consulteu la secció \" {@ book Fer un atac | phb | 9 | fer un atac} \"per a les regles que regulen els atacs.",
```
```suggestion
				"Amb aquesta acció, feu un atac a cos a cosa o a distància. Consulteu la secció \" {@ book Fer un atac | phb | 9 | fer un atac} \"per a les regles que regulen els atacs.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -58,50 +59,50 @@
 		},
 		{
-			"name": "Dash",
+			"name": "Moviement ràpid (dash)",
```
He estat fent servir Córrer, que també la fem servir a 3.5

daniel-giralt-len commented on data/actions.json:
```diff
@@ -147,11 +148,11 @@
 			],
 			"entries": [
-				"If you take the Disengage action, el teu moviment no provoca atacs d'oportunitat for the rest of the turn."
+				"Si feu l'acció de destravar-se, el vostre moviment no provoca atacs d'oportunitat durant la resta del torn."
```
```suggestion
				"Si fas l'acció de destravar-se, el teu moviment no provoca atacs d'oportunitat durant la resta del torn."
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -162,11 +163,11 @@
 			],
 			"entries": [
-				"When you take the Dodge action, you focus entirely on avoiding attacks. Until the start of your next turn, any attack roll made against you has disadvantage if you can see the attacker, and you make Destresa saving throws with advantage. You lose this benefit if you are incapacitated (as explained in the appendix) or if your speed drops to 0."
+				"Quan feu l'acció d'esquivar, us centreu completament a evitar atacs. Fins a l'inici del proper torn, qualsevol tirada d'atac realitzat contra vosaltres té desavantatge si podeu veure l'atacant i podeu fer les tirades de salvació per Destresa amb avantatge. Perdeu aquest benefici si esteu incapacitats (tal com s'explica a l'apèndix) o si la vostre velocitat es redueix a 0. "
```
```suggestion
				"Quan fas l'acció d'esquivar, et centres completament a evitar atacs. Fins a l'inici del teu proper torn, qualsevol tirada d'impacte contra tu té desavantatge si pots veure l'atacant; també pots fer les tirades de salvació per Destresa amb avantatge. Perds aquest benefici si estàs incapacitat (tal com s'explica a l'apèndix) o si la teva velocitat es redueix a 0. "
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -177,23 +178,23 @@
 			],
 			"entries": [
-				"If a spell must be maintained with concentration, that fact appears in its Duration entry, and the spell specifies how long you can concentrate on it. You can end concentration at any time (no requereix acció)."
+				"Si un conjur s'ha de mantenir amb concentració, s'especifica a la seva durada, i el conjur especifica quant de temps podeu concentrar-vos-hi. Podeu finalitzar la concentració en qualsevol moment (no es requereix acció)."
```
```suggestion
				"Si un conjur ha de mantenir amb concentració s'especifica a la seva durada; també especifica quant de temps pots concentrar-t'hi. Pots acabar la concentració en qualsevol moment (no es requereix acció)."
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -204,14 +205,14 @@
 			],
 			"entries": [
-				"A {@condition grappled} creature can use its action to escape. To do so, it must succeed on a Força ({@skill Atletisme}) or Destresa ({@skill Acrobàcies}) check contested by your Força ({@skill Atletisme}) check."
+				"Una criatura {@condition grappled} pot utilitzar la seva acció per fugir. Per fer-ho, ha de tenir èxit en una prova de Força ({@skill Atletisme}) o Destresa ({@skill Acrobàcies}) enfrontada a la Força ({@skill) Atletisme} de qui fa la presa). "
```
```suggestion
				"Una criatura {@condition grappled} pot utilitzar la seva acció per fugir. Per fer-ho, ha de superar una tirada de Força ({@skill Atletisme}) o Destresa ({@skill Acrobàcies}) contra a la Força ({@skill) Atletisme} de qui fa la presa). "
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -222,38 +223,38 @@
 			"entries": [
-				"When you want to grab a creature or wrestle with it, you can use the {@action Attack} action to make a special melee attack, a grapple. If you're able to make multiple attacks with the {@action Attack} action, this attack replaces one of them. The target of your grapple must be no more than one size larger than you, and it must be within your reach.",
-				"Using at least one free hand, you try to seize the target by making a grapple check, una tirada de Força ({@skill Atletisme}) contested by the target's Força ({@skill Atletisme}) or Destresa ({@skill Acrobàcies}) check (the target chooses the ability to use). You succeed automatically if the target is {@condition incapacitated}. If you succeed, you subject the target to the {@condition grapp ...
+				"Quan vulgueu retenir una criatura o lluitar-hi agafats, podeu utilitzar l'acció {@action Atac} per fer un atac a melé especial, una presa. Si podeu fer diversos atacs amb l'acció {@action Atac} , aquest atac substitueix un d'ells. L'objectiu de la vostra presa no pot ser més d'una mida més gran que vosaltres i ha d'estar al vostre abast.",
```
```suggestion
				"Quan vulguis retenir una criatura o lluitar-hi agafats, pods utilitzar l'acció {@action Atac} per fer un atac cos a cos especial, una presa. Si podts fer diversos atacs amb l'acció {@action Atac} , aquest atac substitueix un d'ells. L'objectiu de la vostra presa no pot ser més d'una mida més gran que tu i ha d'estar al teu abast.",
```

kaitokidi replied:
pods xd

daniel-giralt-len commented on data/actions.json:
```diff
@@ -222,38 +223,38 @@
-				"When you want to grab a creature or wrestle with it, you can use the {@action Attack} action to make a special melee attack, a grapple. If you're able to make multiple attacks with the {@action Attack} action, this attack replaces one of them. The target of your grapple must be no more than one size larger than you, and it must be within your reach.",
-				"Using at least one free hand, you try to seize the target by making a grapple check, una tirada de Força ({@skill Atletisme}) contested by the target's Força ({@skill Atletisme}) or Destresa ({@skill Acrobàcies}) check (the target chooses the ability to use). You succeed automatically if the target is {@condition incapacitated}. If you succeed, you subject the target to the {@condition grapp ...
+				"Quan vulgueu retenir una criatura o lluitar-hi agafats, podeu utilitzar l'acció {@action Atac} per fer un atac a melé especial, una presa. Si podeu fer diversos atacs amb l'acció {@action Atac} , aquest atac substitueix un d'ells. L'objectiu de la vostra presa no pot ser més d'una mida més gran que vosaltres i ha d'estar al vostre abast.",
+				"Utilitzant almenys una mà lliure, intenteu retenir a l'objectiu fent una prova de presa, una tirada de Força ({@skill Atletisme}) enfrontada amb la Força ({@skill Atletisme}) o Destresa ({@skill Acrobàcies)}) de l'objectiu (l'objectiu pot triar quina habilitat utilitzar). Teniu èxit automàticament si l'objectiu està {@condition incapacitated}. Si teniu èxit, l'objectiu passa a estar {@condit ...
```
```suggestion
				"Has de tenir almenys una mà lliure. Proves de subjectar a l'objectiu fent una prova de presa, una tirada de Força ({@skill Atletisme}) enfrontada amb la Força ({@skill Atletisme}) o Destresa ({@skill Acrobàcies)}) de l'objectiu (l'objectiu pot triar quina habilitat utilitzar). La superes automàticament si l'objectiu està {@condition incapacitated}. Si superes, l'objectiu passa a estar {@condition grappled} (vegeu l'apèndix). A la condició s'especifiquen les condicions de finalització. Pots alliberar l'objectiu sempre que vulgueu (no requereix acció). ",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -222,38 +223,38 @@
+					"name": "Moure un Objectiu Apresat",
 					"entries": [
-						"When you move, you can drag or carry the {@condition grappled} creature with you, but your speed is halved, unless the creature is two or more sizes smaller than you."
+						"Quan us moveu, podeu arrossegar o carregar la criatura {@condition grappled} amb vosaltres, però la vostra velocitat es reduirà a la meitat, tret que la criatura sigui dues o més mides més petita que vosaltres."
```
```suggestion
						"Quan et mous, pods arrossegar o carregar la criatura {@condition grappled} amb tu, però la teva velocitat es reduirà a la meitat, tret que la criatura sigui dues o més mides més petita que tu."
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -222,38 +223,38 @@
-				"Com a acció, a character can use a healing surge and spend up to half his or her Hit Dice. For each Hit Die spent in this way, the player rolls the die and adds the character's Constitució modifier. The character regains hit points equal to the total. The player can decide to spend an additional Hit Die after each roll.",
-				"A character who uses a healing surge can't do so again until he or she finishes a short or long rest. Under this optional rule, a character regains all spent Hit Dice at the end of a long rest. With a short rest, a character regains Hit Dice equal to his or her level divided by four (minimum of one dice)."
+				"Com a acció, un personatge pot utilitzar un esforç curatiu i gastar fins a la meitat dels seus daus de cop. Per cada dau de cop gastat d'aquesta manera, el jugador tira el dau i hi suma el seu modificador de Constitució, i el personatge recupera tants punts de cop com el resultat. El jugador pot decidir gastar un dau de cop addicional després de cada tirada.",
+				"Un personatge que utilitza un esforç curatiu no ho pot tornar a fer fins després d'un descans (curt o llarg). Segons aquesta regla opcional, un personatge recupera tots els daus de cop gastats després d'un descans llarg. Amb un descans curt, un personatge recupera daus de cop igual al seu nivell dividit per quatre (mínim d'un dau)."
```
```suggestion
				"Un personatge que utilitza un esforç curatiu no ho pot tornar a fer fins després d'un Descans Curt o Llarg. Segons aquesta regla opcional, un personatge recupera tots els Daus de Vida gastats després d'un descans llarg. Amb un descans curt, un personatge recupera Daus de Vida igual al seu nivell dividit per quatre (mínim d'un dau)."
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -222,38 +223,38 @@
 			"entries": [
-				"Com a acció, a character can use a healing surge and spend up to half his or her Hit Dice. For each Hit Die spent in this way, the player rolls the die and adds the character's Constitució modifier. The character regains hit points equal to the total. The player can decide to spend an additional Hit Die after each roll.",
-				"A character who uses a healing surge can't do so again until he or she finishes a short or long rest. Under this optional rule, a character regains all spent Hit Dice at the end of a long rest. With a short rest, a character regains Hit Dice equal to his or her level divided by four (minimum of one dice)."
+				"Com a acció, un personatge pot utilitzar un esforç curatiu i gastar fins a la meitat dels seus daus de cop. Per cada dau de cop gastat d'aquesta manera, el jugador tira el dau i hi suma el seu modificador de Constitució, i el personatge recupera tants punts de cop com el resultat. El jugador pot decidir gastar un dau de cop addicional després de cada tirada.",
```
```suggestion
				"Com a acció, un personatge pot utilitzar un esforç curatiu i gastar fins a la meitat dels seus Daus de Vida. Per cada Dau de Vida gastat d'aquesta manera, el jugador tira el dau i hi suma el seu modificador de Constitució, i el personatge recupera el resultat en punts de vida. El jugador pot decidir gastar un Dau de Vida addicional després de cada tirada.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -299,38 +300,38 @@
 			"entries": [
-				"Your character can do things not covered by the actions in this chapter, such as breaking down doors, intimidating enemies, sensing weaknesses in magical defenses, or calling for a parley with a foe. The only limits to the actions you can attempt are your imagination and your character's ability scores. See the descriptions of the ability scores in {@book chapter 7|phb|7|Ability Scores and M ...
-				"When you describe an action not detailed elsewhere in the rules, the DM tells you whether that action is possible and what kind of roll you need to make, if any, to determine success or failure."
+				"El vostre personatge pot fer coses que no estan cobertes per les accions d'aquest capítol, com ara trencar portes, intimidar enemics, detectar debilitats en les defenses màgiques o demanar parlamentar amb un enemic. Els únics límits a les accions que podeu intentar són la vostre imaginació i les puntuacions de les habilitats del vostre personatge. Vegeu les descripcions de les puntuacions de ...
```
```suggestion
				"El vostre personatge pot fer coses que no estan cobertes per les accions d'aquest capítol, com ara trencar portes, intimidar enemics, detectar debilitats en les defenses màgiques o demanar parlamentar amb un enemic. Els únics límits a les accions que podeu intentar són la vostre imaginació i les puntuacions de característica del vostre personatge. Vegeu les descripcions de les puntuacions de les habilitats a {@book capítol 7 | phb | 7 | Puntuacions i modificadors de les habilitats} per inspirar-vos mentre improviseu.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -299,38 +300,38 @@
 			"entries": [
-				"This option makes it easier for melee combatants to harry each other with opportunity attacks.",
-				"When a creature makes a melee attack, it can also mark its target. Until the end of the attacker's next turn, any opportunity attack it makes against the marked target has advantage. The opportunity attack doesn't expend the attacker's reaction, but the attacker can't make the attack if anything, such as the {@condition incapacitated} condition or the {@spell shocking grasp} spell, is preven ...
+				"Aquesta opció facilita als combatents a melé atacar-se amb atacs d'oportunitat.",
```
```suggestion
				"Aquesta opció facilita als combatents a cos a cos atacar-se amb atacs d'oportunitat.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -341,65 +342,65 @@
+				"Podeu comunicar-vos com us sigui possible, mitjançant breus enunciats i gestos, mentre preneu el vostre torn.",
+				"També podeu interactuar de forma gratuïta amb un objecte o característica de l'entorn, ja sigui durant el vostre moviment o la vostra acció. Per exemple, podeu obrir una porta durant el vostre moviment mentre camineu cap a un enemic o podeu desenfundar l'arma com a part de la mateixa acció que utilitzeu per atacar. ",
+				"Si voleu interactuar amb un segon objecte, heu d'utilitzar la vostra acció. Alguns Ítems màgics i altres ítems especials sempre requereixen una acció, tal com s'indica a les seves descripcions.",
+				"El MM pot requerir que utilitzeu una acció per a qualsevol d'aquestes activitats quan necessiti una cura especial o quan presenti un obstacle inusual. Per exemple, el MM pot requerir raonablement que utilitzeu una acció per obrir una porta travada o girar una manovella per baixar un pont llevadís.",
```
```suggestion
				"El/la/li MM pot requerir que utilitzeu una acció per a qualsevol d'aquestes activitats quan necessiti una cura especial o quan presenti un obstacle inusual. Per exemple, el/la/li MM pot requerir raonablement que utilitzeu una acció per obrir una porta travada o girar una manovella per baixar un pont llevadís.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -429,15 +430,15 @@
-				"When you ready a spell, you cast it as normal but hold its energy, which you release with your reaction when the trigger occurs. To be readied, a spell must have a casting time of 1 action, and holding onto the spell's magic requires concentration (explained in {@book chapter 10|phb|10|concentration}). If your concentration is broken, the spell dissipates without taking effect. For example,  ...
-				"You have until the start of your next turn to use a readied action."
+				"A vegades voleu saltar sobre un enemic o esperar una circumstància particular abans d'actuar. Per fer-ho, podeu fer l'acció Preparada al vostre torn, que us permet actuar fent servir la vostra reacció abans de començar el següent torn.",
+				"En primer lloc, decidiu quina circumstància perceptible desencadenarà la vostra reacció. Després, trieu l'acció que voleu executar en resposta a aquest activador o decidireu moure-us fins a la vostra velocitat per respondre-hi. Els exemples inclouen \" Si els enemics trepitgen la zona de la trampa, estiraré la palanca que l'obre, \" i \" Si el gòblin passa al meu costat, m'allunyo. \"",
```
```suggestion
				"En primer lloc, decidiu quin detonant (perceptible!) desencadenarà la vostra reacció. Després, trieu l'acció que voleu executar en resposta a aquest detonant; també pots moure't fins a la vostra velocitat per respondre-hi. Els exemples inclouen \" Si els enemics trepitgen la zona de la trampa, estiraré la palanca que l'obre, \" i \" Si el gòblin passa al meu costat, m'allunyo. \"",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -429,15 +430,15 @@
-				"You have until the start of your next turn to use a readied action."
+				"A vegades voleu saltar sobre un enemic o esperar una circumstància particular abans d'actuar. Per fer-ho, podeu fer l'acció Preparada al vostre torn, que us permet actuar fent servir la vostra reacció abans de començar el següent torn.",
+				"En primer lloc, decidiu quina circumstància perceptible desencadenarà la vostra reacció. Després, trieu l'acció que voleu executar en resposta a aquest activador o decidireu moure-us fins a la vostra velocitat per respondre-hi. Els exemples inclouen \" Si els enemics trepitgen la zona de la trampa, estiraré la palanca que l'obre, \" i \" Si el gòblin passa al meu costat, m'allunyo. \"",
+				"Quan es produeix el disparador, podeu executar la vostra reacció just després d'acabar el disparador o ignorar-lo. Recordeu que només podeu fer una reacció per ronda.",
```
```suggestion
				"Quan es s'activa el detonador, podeu executar la vostra reacció just després d'acabar el disparador o ignorar-lo. Recordeu que només podeu fer una reacció per ronda.",
```

daniel-giralt-len commented on data/actions.json:
```diff
@@ -501,14 +502,14 @@
 			],
 			"entries": [
-				"A creature can try to tumble through a hostile creature's space, ducking and weaving past the opponent. As an action or a bonus action, the tumbler makes una tirada de Destresa ({@skill Acrobàcies}) contested by the hostile creature's Destresa ({@skill Acrobàcies}) check. If the tumbler wins the contest, it can move through the hostile creature's space once this turn."
+				"Una criatura pot provar de fer malabarismes per passar per l'espai d'una criatura hostil, esquivant-lo i passant de llarg de l'oponent. Com a acció o acció addicional, podeu fer una tirada de Malabarismes tirant Destresa ({@skill Acrobàcies}) enfrontada amb la Destresa ({ @skill Acrobàcies}) de la criatura hostil. Si guanyeu l'enfrontament, podeu moure-us una vegada per l'espai de la criatur ...
```
```suggestion
				"Una criatura pot provar de fer piruetes per passar per l'espai d'una criatura hostil, esquivant-lo i passant de llarg de l'oponent. Com a acció o acció bonus, pots fer una tirada Destresa ({@skill Acrobàcies}) contra la Destresa ({@skill Acrobàcies}) de la criatura hostil. Si guanyeu l'enfrontament, podeu moure-us una vegada per l'espai de la criatura hostil aquest torn."
```

Pels vells temps :3

daniel-giralt-len reviewed (commented):
Revisat! Coses en general que no he afegit que queden
- 3a persona del singular > 1a persona del singular
- melé > cos a cos
- rotllo > pergamí

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
cleaning

kaitokidi committed:
:wq
Merge branch 'actionsNDreplacements' of github.com:kaitokidi/5ein.es into actionsNDreplacements

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
Update data/actions.json

Co-authored-by: Daniel Giralt Len <daniel.giralt.len@gmail.com>

kaitokidi committed:
item to object

kaitokidi committed:
object to item back

kaitokidi committed:
merge

kaitokidi committed:
readed the search and replaces

kaitokidi committed:
last item to object
